.github/workflows/client.yml

```
@@ -44,7 +46,7 @@ jobs:
   client-build-test-publish-ethereum:
     needs: client-detect-changes
     if: |
-      github.event_name == 'push'
+      github.event_name != 'pull_request'
```

We don't want to execute the job for example for `workflow_dispatch` with `alfajores` input. With the proposed config, the job would execute for such workflow.

My suggestion:
```suggestion
      github.event_name == 'push'
        || github.event_name == 'schedule'
```

---

agree, good catch c534da4a963b8b428d8936a9ff7e9ca0c54f1983